src/Button/Button.js

```diff
@@ -291,5 +291,17 @@ module.exports = kind(
 		if (opacity == 'translucent' || opacity == 'transparent') {
 			this.addClass(opacity);
 		}
-	}
+	},
+
+	// Accessibility
+
+	/**
+	* @private
+	*/
+	ariaObservers: [
+		{path: ['content'], method: function () {
+			var content = this.content;
+			this.$.client.setAriaAttribute('aria-label', content);
```

Comment:
Does it guarantee timing in case of developer set accessibility label and content dynamically?

ex 1) if we couldn't guarantee this, we need to consider reference guide that developers should use following sequence reversely like ex2.
`
this.set('accessibilityLabel', 'a11y'); // we should pick this to use a11y label prior than following content
`
`
this.set('content', 'normal');
`

ex 2) I think it's ok for this case, but need to check
`
this.set('content', 'normal');
`
`
this.set('accessibilityLabel', 'a11y'); // we should pick this to use a11y label prior than following content
`